live-examples/js-examples/regexp/regexp-quantifiers.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const ghostSpeak = "booh boooooooh";
+const regexpSpooky = /bo{3,}h/;
+console.log(ghostSpeak.match(regexpSpooky));
+// expected output: Array ["boooooooh"]
+
+let htmlSnippet = "&lt;span foo='bar' baz&gt;Hello&lt;span/&gt;"
```

Comment:
let htmlSnippet = "&lt;span foo='bar' baz&gt;Hello&lt;span/&gt;"
```suggestion
const htmlSnippet = "&lt;span foo='bar' baz&gt;Hello&lt;span/&gt;";
```

Reply:
Agreed with this. I was too quick to apply my "mdn-linting-fixing with regexp" own procedure to create an example. I'll figure out another one :)